tests/unit/test-class-wc-payments-account.php

```diff
@@ -428,7 +428,9 @@ function () {
 		);
 
 		// Act.
+		ob_start();
```

Comment:
I suggest we add an explanatory comment here: 

```suggestion
        // Some code paths in the onboarding output JSON, so we use output buffering to suppress it while testing.
		ob_start();
```